### ISSUE TITLE
Project Title: Accessibility Roles and Responsibilities Mapping (ARRM) ? [was: Why "Matrix"?]

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-roles-responsbilities#10]
To me, the main deliverable is a **simple table** showing Roles and Responsibilities for WCAG 2 Success Criteria. I'm missing why we would call it a "matrix".

(Also, I'm not sure how well "matrix" translates.)

I looked through definitions of "matrix" and it doesn't seem to fit...

**Comment by bakkenb** [comment]
Well... After looking up the word "matrix," I would have to agree with Shawn here. I have been under the impression, I guess my whole life, that a matrix is a sort of table that has data. Based on definition, it is not. I would agree that it will not translate well either. 

Thanks Shawn, I have learned the proper definition of an old vocabulary word I thought I knew. :(

**Comment by orvalle** [comment]
I'm not dedicated to preserving the term unless there's more commitment to it.

**Comment by dboudreau** [comment]
@slhenry - I hear what you're saying, and though I find "a simple table" a little offensive (no harm done), I can understand that matrix might not be the most appropriate term to use. We've looked it up (aside from all Keanu Reeves references), but it's more than just that. We think of it as a blueprint or a framework at the very least, that empowers an organization to tackle accessibility and break it down into their processes. Beyond the simple table, there are tools that come with this, and elements of a process. 

We feel that a word like matrix translates better to other languages that framework or blueprint might, hence our preference with using that term. According to dictionary.com, a matrix is:

**matrix**
[mey-triks, ma-]
noun, plural ma·tri·ces [mey-tri-seez, ma-] /ˈmeɪ trɪˌsiz, ˈmæ-/, ma·trix·es.
something that constitutes the place or point from which something else originates, takes form, or develops.

We look at part of this project as a RACI matrix. You'll notice that the agreed upon term here is matrix, while it is very much just a table. But the data in that table has different layers of meaning and value, and that makes it more complex than a simple two-dimensional representation of data. It's complexity brings it at another level and it's also what we want to convey with this project.

So to me, "matrix" kind of fits, but I don't want to be pendantic about it either. That said, we are not married to the term, but it has an all encompassing feel to it that we like better. From the perspective of how it can be perceived, "matrix" is not used to death the same way "framework" would be in developer circles, so we like it better for that reason, too.

If people think a better term could be used, we're happy to receive proposals. It's not too late to change it.

**Comment by shawna-slh** [comment]
Yes, yes. The work involved in developing it is *much* more than a simple table. I don't propose that we call it that!

**Comment by shawna-slh** [comment]
Thanks for the thoughts! (I agree framework and blueprint aren't as good.)

I'd be OK with "Matrix" if others are and we don't come up with something better.

**UPDATE based on EOWG 15 June 2018, +1 for any of these:**

- WARM - Web Accessibility Roles Mapping
- WARM - Web Accessibility Responsibilities Mapping
- WARM - Web Accessibility Roles and Responsibilities Mapping

(I think it's OK that it includes Web and Mobile)
Or, maybe OK to leave off "Web"

- ARM - Accessibility Roles and Responsibilities Mapping

/me thought about Digital Accessibility... but DARM not so good. :-) 

(As I said previously, I'm not comfortable with "RA11y" -- and was told that's just a working title and it will be reconsidered later.)

**Comment by dboudreau** [comment]
Norah suggested "Accessibility Role Mapping"... mapping has a proactive ring to it.

**Comment by shawna-slh** [comment]
[EOWG discussion from 15 June 2018 telecon](https://www.w3.org/2018/06/15-eo-minutes.html#item02)

**Comment by dboudreau** [comment]
After some thought, we've looked into ARM as an acronym, and found quite a bit already... we probably shouldn't go down that route, unless we want to fight for visibility in search engines... there's a lot of ARMs out there - 26 usages, several are common in specific domains (132 total): https://www.acronymfinder.com/Business/ARM.html

But ARRM has a lot less (only 2 in business, 6 in total): https://www.acronymfinder.com/Business/ARRM.html. We may be on to something, here.

**Comment by dboudreau** [comment]
Voting for **ARRM** (Accessibility Roles and Responsibilities Mapping) Matrix as the new acronym to define the project, instead of "ra11y matrix". Does anyone agree?

**Comment by RachelComerford** [comment]
Are we allowed to talk about ARMing our teams with all the resources they need? :)

**Comment by shawna-slh** [comment]
How about more simply:

**Accessibility Roles and Responsibilities Mapping (ARRM)**

(w/o "Matrix", which seems unnecessary and a bit redundant with "Mapping")?

That seems to cover the issues that you want -- that it's more than a static table - it's an active mapping , that you can customize for your organization! :-)

**Comment by dboudreau** [comment]
Yup, let's go with Accessibility Roles and Responsibilities Mapping (ARRM). 

I'll make the changes in the project plan document, though I'll leave the project code name to "ra11y matrix", just because of the wiki page name/URL.
